R/nhanes_tables.R

```diff
@@ -157,6 +161,11 @@ nhanesTableVars <- function(data_group, nh_table, details = FALSE, nchar=128, na
     stop("Invalid survey group")
     return(NULL)
   }
+
+  if(!is.na(collection_date) & !is.na(container_version)){
```

Comment:
Would you recommend changing the default value of nchar? It's currently 128.

Reply:
I would try to make it as long as possible - for most things 256 seems to be enough - I am just looking at the database schema and we use varchar(256) a lot.  There are a couple of places where it gets bigger - 1024 - but I don't think for any of the variables or their descriptions.

Reply:
I can change it in the next release